### PR TITLE
Add docs to persist Grafana dashboards

### PR DESCRIPTION
**Pull request by catherineluse**
https://github.com/rancher/docs/issues/2805

### CODE REVIEW

**Review by aiyengar2 — APPROVED** [review]
LGTM, @bashofmann if you could take a look that would be great.

Should be able to merge either way though.

File: content/rancher/v2.x/en/monitoring-alerting/v2.5/persist-grafana/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. In the left navigation menu, click **JSON Model.**
+1. Copy the JSON data structure that appears.
+1. Create a ConfigMap in the `cattle-dashboards` namespace. The ConfigMap needs to have the label `grafana_dashboard: "1"`. Paste the JSON into the ConfigMap in the format shown in the example below:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      labels:
+    	grafana_dashboard: "1"
```
Comment (bashofmann) [inline content/rancher/v2.x/en/monitoring-alerting/v2.5/persist-grafana/_index.md:25]: This must be indented two additional spaces to the right.

**Review by bashofmann — CHANGES_REQUESTED** [review]
Looks good, apart from the indentation of the label.

File: content/rancher/v2.x/en/monitoring-alerting/v2.5/persist-grafana/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/monitoring-alerting/v2.5/persist-grafana/_index.md]: changed it to:

```
apiVersion: v1
    kind: ConfigMap
    metadata:
      labels:
        grafana_dashboard: "1"
      name: <dashboard-name>
      namespace: cattle-dashboards
    data:
      <dashboard-name>.json: |-
        <copied-json>
```

**Pull request merged**